doc/api/cli.md

```diff
@@ -184,6 +184,15 @@ added: v11.8.0
 
 Enable experimental diagnostic report feature.
 
+### `--source-map-support`
```

Comment:
e anyI’d go with `--source-maps` or `--enable-source-maps`… I don’t hav strong arguments for that, besides maybe that flags usually imply support for some kind of feature already?

Reply:
I like `--enable-source-maps`, personally (I've updated to this and will push soon).